requests_toolbelt/adapters/appengine.py

```diff
@@ -27,6 +31,7 @@ class AppEngineAdapter(adapters.HTTPAdapter):
         >>> from requests_toolbelt.adapters import appengine
         >>> s = requests.Session()
         >>> if using_appengine():
```

Comment:
Not sure, this docstring predates me in @sigmavirus24 's original pull request. I'm personally not a fan of this approach anyway, preferring the monkeypatch() approach below, so that my third-party libraries I use automatically get working Sessions.